backend/LearningCalendar/Epicenter.Persistance/Repository/LearningCalendar/EmployeeRepository.cs

```diff
@@ -87,5 +87,15 @@ public async Task<List<Employee>> GetByTopicIdAsync(Guid topicId)
                 .ToListAsync();
             return result;
         }
+
+        public async Task<Employee> GetWithSubordinateTree(Guid id)
+        {
+            var result = await DbContext.Employees
+                .Include(employee => employee.Team)
+                .ThenInclude(team => team.Manager)
+                .ToListAsync();

```

Comment:
It's hard for me to believe that this code returns employee `WithSubordinateTree` as said in the name 😄. Looks like it's going up and not down, which (I think) is never required in our system

Reply:
Mhm, will remove this, thanks